src/scripts/breeding/Egg.ts

```diff
@@ -143,12 +143,14 @@ class Egg implements Saveable {
                 partyPokemon.level = 1;
                 partyPokemon.breeding = false;
                 partyPokemon.level = partyPokemon.calculateLevelFromExp();
-                if (partyPokemon.pokerus == GameConstants.Pokerus.Infected) {
-                    partyPokemon.pokerus = GameConstants.Pokerus.Contagious;
-                }
-                if (partyPokemon.evs() >= 50 && partyPokemon.pokerus == GameConstants.Pokerus.Contagious) {
-                    partyPokemon.pokerus = GameConstants.Pokerus.Resistant;
-                }
+            }
+
+            // Update pokerus status
+            if (partyPokemon.pokerus == GameConstants.Pokerus.Infected) {
+                partyPokemon.pokerus = GameConstants.Pokerus.Contagious;
+            }
+            if (partyPokemon.evs() >= 50 && partyPokemon.pokerus == GameConstants.Pokerus.Contagious) {
+                partyPokemon.pokerus = GameConstants.Pokerus.Resistant;
```

Comment:
I know it was already there, but is this even needed anymore with the `evs` subscriber in `PartyPokemon`?